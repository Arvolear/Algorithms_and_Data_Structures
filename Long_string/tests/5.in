2975808881599380081372348
11968165665773824772772002536387573311090567