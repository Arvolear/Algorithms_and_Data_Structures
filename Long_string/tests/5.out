11968165665773824775747811417986953392462915
11968165665773824769796193654788193229718219
35614973684662505606781988705240107898466183812939269394033877441316
4021819324412193787